code/__init__.py

```diff
@@ -0,0 +1 @@
+from code.code import armstrong
```

Comment:
I don't think we need this to actually do anything. It's just a way to tell python "this is a module".

https://docs.python.org/3/tutorial/modules.html#packages

Reply:
Yes, it is ok if we keep the __init__ file empty for this repository. This was just to follow the custom of mentioning the modules in the init.py.